console/docable.ts

```diff
@@ -0,0 +1,91 @@
+const decoder = new TextDecoder();
+const encoder = new TextEncoder();
+
+/**
+ * Parse data member doc blocks and signatures and place them in a minimalistic,
+ * JSON format.
+ */
+export class Docable {
+
+  protected filepaths: string[];
+  protected json_output: {[k: string]: string[]} = {};
+  protected output_filepath: string;
+
+  constructor(filepaths: string[], outputFilepath: string) {
+    this.filepaths = filepaths;
+    this.output_filepath = outputFilepath;
+  }
+
+  public async run() {
+    for (let index in this.filepaths) {
```

Comment:
feel like `for (const filepath of this.filepaths)` might be better? nitpicky though

Reply:
3604b6c

Reply:
wrong commit sha but i see you made the change lol